### ISSUE TITLE
Touch screen on mobile demo

### ISSUE THREAD

**Issue by anvaka** [rowanwins/visibility-graph#1]
Hi, Rowan!

Thank you for the implementation and very nice demo! When I was on mobile phone (iPhone) I couldn't tell what this library is doing because I was trying to drag the "pin-point" icons.

It would be awesome to have the demo mobile friendly :).

**Comment by rowanwins** [comment]
Yeah mobile friendly would be nice although admittedly it's kinda low priority - still got one or two bugs to iron out before moving on to gold-plating :)
